ui/common/src/commonMain/kotlin/social/androiddev/common/composables/Spacers.kt

```diff
@@ -0,0 +1,64 @@
+/*
+ * This file is part of MastodonX.
+ *
+ * MastodonX is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+ *
+ * MastodonX is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with MastodonX. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package social.androiddev.common.composables
+
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.RowScope
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun RowScope.WeightSpacer(modifier: Modifier = Modifier) {
+    Spacer(modifier = modifier.weight(1f))
+}
+
+@Composable
+fun ColumnScope.WeightSpacer(modifier: Modifier = Modifier) {
+    Spacer(modifier = modifier.weight(1f))
+}
+
+@Composable
+fun XSmallSpacer(modifier: Modifier = Modifier) {
+    Spacer(modifier = modifier.size(2.dp))
```

Comment:
Agree with @exallium comment, and grouping these under a namespace really helps with discoverability when trying look up our component spacers. 

I was planning on doing something similar for common paddings like

```kotlin
object Spacing {
       val XS = 4.dp
       val S = 6.dp
       val L = 12.dp
       ...
}
```

Reply:
Potentially yes, also I just wrote up a quick example. Updated my original comment with a different one that I think is more fitting.

And regardless, the discoverability comment still applies in general.